controllers/transactionControllers.js

```diff
@@ -48,7 +51,11 @@ export const createTransaction = async (req, res) => {
       amount,
     });
 
-    res.status(201).json(newTransaction);
+    res.status(201).json({
+      status: 'Created',
+      code: 201,
+      data: { 'Added new transaction:': newTransaction },
```

Comment:
Tak samo jak powyżej, jak chcesz dodać jakiś komunikat to może być oddzielny message: "Added new transaction"